tests/test_tutorial/test_exceptions/test_tutorial005.py

```diff
@@ -0,0 +1,34 @@
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+from typer.testing import CliRunner
+
+from docs_src.exceptions import tutorial005 as mod
+
+runner = CliRunner()
+
+
+def test_traceback_rich_pretty_width():
```

Comment:
This test simply checks that the program runs like expected, but it doesn't (yet) actually test the Rich borders length as I wasn't quite sure how to do that 🤔